### ISSUE TITLE
Water breathing icon shows

### ISSUE THREAD

**Issue by stayandwait** [voxelwind/voxelwind#46]
The water breathing icon is showing but I am not underwater.

![2016-10-31_1824](https://cloud.githubusercontent.com/assets/15825041/19873835/9dd363b6-9f97-11e6-8ef2-75ee4c2ca361.png)

**Comment by minecrafter** [comment]
Known issue, haven't found a resolution yet. Will look into this.

**Comment by mhsjlw** [comment]
Not sending metadata properly afaik

**Comment by TheDiamondYT1** [comment]
I can't join using 1.0.0

**Comment by TheDiamondYT1** [comment]
I changed protocol version to 100 and changed the raknet broadcast details

**Comment by mhsjlw** [comment]
Yeah, good luck. It's going to require rewriting the chunk format and changing various packets along with the login sequence

**Comment by TheDiamondYT1** [comment]
In branch 1.0 it says fully working

**Comment by mhsjlw** [comment]
This is outside the scope of this issue
